src/components/ecology.jsx

```diff
@@ -3,42 +3,44 @@ import API from "./api";
 import Overview from "./overview";
 
 export default class Ecology extends React.Component {
-  renderAPI(source) {
```

Comment:
My understanding is this method is more functional, making it more desirable and easier to parse... but it looks like it needs two changes:
`<API source={source} />`
and some kind of `else return;` line if `source` doesn't exist or, alternatively, an early exit, `if(!source) return;`.

@coopy Is this correct?